src/web/mod.rs

```diff
@@ -57,20 +55,18 @@ impl WebError {
     /// Otherwise, constructs [`WebError::Internal`].
     fn expect_iroha_find_error(client_error: IrohaClientQueryError) -> Self {
         match client_error {
-            IrohaClientQueryError::QueryError(IrohaQueryError::Find(_err)) => Self::NotFound,
-            IrohaClientQueryError::QueryError(other) => {
-                Self::Internal(eyre!("FindError expected, got: {other}"))
-            }
+            IrohaClientQueryError::Validation(_err) => Self::NotFound,
+
             IrohaClientQueryError::Other(other) => {
-                Self::Internal(other.wrap_err("Unexpected query error"))
+                Self::Internal(other.wrap_err("Unexpected query error:{other}"))
```

Comment:
```suggestion
                Self::Internal(other.wrap_err("Unexpected query error: {other}"))
```

Please pay more attention to appropriate spacing. I noticed it a lot of places, including DMs, comments, PR description etc.

Reply:
@horizenight, you have put thumb-up, resolved the conversation, but haven't applied it.